packages/ui/src/helpers/shared/validator.ts

```diff
@@ -0,0 +1,42 @@
+import { PasswordSettings } from '../../types';
+import { hasSpecialChars } from '../authenticator/utils';
+
+/** Checks password against password settings from `Amplify.configure()`  */
+export const defaultPasswordValidator = (
+  password: string,
+  passwordSettings: PasswordSettings
+): string[] => {
+  const errors = [];
+
+  const policyMinLength = passwordSettings?.passwordPolicyMinLength;
+  if (password.length < policyMinLength) {
+    errors.push(`Password must have at least ${policyMinLength} characters`);
+  }
```

Comment:
These logics are migrated from `defaultServices.validateFormPassword`, minus Authenticator specific classnames or blur logic.

Reply:
Was all this moved just to clean up the `defaultServices` logic?

Reply:
I see and help with duplication of logic when the Account Services checks go into place. 👍

Reply:
Right, I should've added more to PR description 😓 

This is specifically for `ChangePassword` component, which will have almost the same validation logic as Authenticator one.